<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NovoProduto</class>
 <widget class="QWidget" name="NovoProduto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Novo Produto</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icons/mdpi.png</normaloff>../Icons/mdpi.png</iconset>
  </property>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="tbNome">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalvar">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="tbDescricao">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Descrição:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbFornecedor">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Fornecedor:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
